tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md

```diff
@@ -87,5 +87,4 @@ launchers:
       batch: 4
     mo_flags:
       - reverse_input_channels
-    cpu_extensions: cpu_extentions_avx512.so
```

Comment:
we not remove functional opportunity to set extension, so I do not think that we should remove it from docs sample (as I know, it is still used for example for running on arm cpu)

Reply:
I suppose I could keep the key, but what do I use for the value? The `cpu_extensions_avx512` library doesn't exist anymore.

Reply:
cpu_extention.so or custom_cpu_extension.so will be ok (it is just abstract name used in example purposes, for running with extension, of course, user should compile them on own side)

Reply:
Alright, updated.